youtube_dl/extractor/mitele.py

```diff
@@ -91,3 +87,37 @@ def _real_extract(self, url):
             'age_limit': int_or_none(info.get('rating')),
             'timestamp': parse_iso8601(pre_player.get('publishedTime')),
         }
+
+        if video.get('dataCmsId') == 'ooyala':
+            info.update({
+                '_type': 'url_transparent',
+                # for some reason only HLS is supported
+                'url': smuggle_url('ooyala:' + video_id, {'supportedformats': 'm3u8,dash'}),
+            })
+        else:
+            config = self._download_json(
+                video['dataConfig'], video_id, 'Downloading config JSON')
+            services = config['services']
+            gbx = self._download_json(
+                services['gbx'], video_id, 'Downloading gbx JSON')
+            caronte = self._download_json(
+                services['caronte'], video_id, 'Downloading caronte JSON')
+            cerbero_data = {
+                'bbx': caronte['bbx'],
+                'gbx': gbx['gbx']
+            }
+            headers = {
+                'Content-Type': 'application/json;charset=UTF-8',
+                'Origin': 'https://www.mitele.es'
+            }
```

Comment:
Inline.

Reply:
I made like this for improving readability from my side as I read in the documentation, but I understand your point of view.